test/torchaudio_unittest/librosa_compatibility_test.py

```diff
@@ -165,13 +165,23 @@ def _load_audio_asset(*asset_paths, **kwargs):
 @unittest.skipIf(not LIBROSA_AVAILABLE, "Librosa not available")
 class TestTransforms(common_utils.TorchaudioTestCase):
     """Test suite for functions in `transforms` module."""
-    def assert_compatibilities(self, n_fft, hop_length, power, n_mels, n_mfcc, sample_rate):
+
+    @staticmethod
+    def _set_up_sound():
         common_utils.set_audio_backend('default')
         path = common_utils.get_asset_path('sinewave.wav')
         sound, sample_rate = common_utils.load_wav(path)
```

Comment:
Could you 
1. Instead of loading data from file, generate one?
    ```python
    sample_rate = 16000
    sound = common_utils.get_sinusoid(n_channels=1, sample_rate=sample_rate)
    ```
2. Instead of making a common method, put the above code in each test method?

These points were not included in the original issue description, but also important and I would like to take advantage of this opportunity to address them.

The first point is to make the unit test not depend of other components. When our load function has some issue, this test could also fail, which makes it harder to see what's going on when looking at log. Unit test should be simple and independent. The second point is the extension of independency. Having common method often makes new contributors think that they have to use the same method for some reasons they do not know. If it's related to the core of the test logic, it is more beneficial to make fixture common, but in this case it is not, so making each unit test independent each other gives better maintenance experience.